.editorconfig

```diff
@@ -26,6 +26,16 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 spelling_exclusion_path = spelling.dic
+dotnet_style_coalesce_expression = true:warning
```

Comment:
These config are already included in this file.

Reply:
I know, but just opening the file in VS adds this